packages/test/test-end-to-end-tests/src/test/gc/gcSweepUnreferencePhases.spec.ts

```diff
@@ -165,17 +169,38 @@ describeCompat("GC unreference phases", "NoCompat", (getTestObjectProvider) => {
 			"Datastore should be tombstoned",
 		);
 
-		// Wait sweepGracePeriodMs, triggering Sweep
+		// Stage 3 - Tombstone -> SweepReady //
+
+		// Wait sweepGracePeriodMs, triggering GC op with the sweep ready nodes.
 		await delay(sweepGracePeriodMs);
 		mainDataStore._root.set("send", "op2");
+		// Close the main container before sweep so that it doesn't end up deleting local data stores which
```

Comment:
Does this need to happen here already?  I would think it could wait until right before the `ensureSynchronized` that transmits the GC op (line 196)